src/Components/WatchlistTile.js

```diff
@@ -53,15 +53,17 @@ export default function WatchlistTile({
           >
             {name}
           </Typography>
-          <Typography
-            variant="body2"
-            sx={{
-              marginLeft: "8px",
-              flexShrink: 0,
-            }}
-          >
-            {items?.length ?? "0"} {items?.length === 1 ? "item" : "items"}
-          </Typography>
+          {items && (
+            <Typography
+              variant="body2"
+              sx={{
+                marginLeft: "8px",
+                flexShrink: 0,
+              }}
+            >
+              {items?.length ?? "0"} {items?.length === 1 ? "item" : "items"}
```

Comment:
nit: `items` should now be guaranteed to have a truthy value here, so the `?` in `items?` are unneeded

Reply:
Great catch!!